XamarinTestRunner/XamarinTestRunner/PlayFabSDK/PlayFabSettings.cs

```diff
@@ -50,19 +50,21 @@ public class PlayFabSettings
 #endif
         #endregion Deprecated staticSettingsredirect properties
```

Comment:
nitpick: Should the redirect be capitalized?

Reply:
Not sure chief but I think since it's not directly related to the context here we can make a separate PR for this later.